src/App.jsx

```diff
@@ -14,70 +15,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [food, setFood] = useState('Jam');
```

Comment:
It would be better to write it like this, because it's not obvious that it's exactly the selected item
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```